actual-reqs.txt

```diff
@@ -0,0 +1,128 @@
+# This file contains almost all the dependencies for ironic.
+# Some are needed, some are not, let's hack out all possible.
+# Some are missing on purpose as they can be installed using rpms
+# also available downstream and they should be added to Dockerfile.
+#
+# All commented out are downstream names.
+# We can specify versions if needed.
+#
+alembic
+amqp
+appdirs
+asn1crypto
+automaton
+cachetools
+certifi
+cffi
+chardet
+cliff
+cmd2
+colorama
+cryptography
+decorator
+#dns
+dnspython
+dogpile-cache
+eventlet
+extras
+fasteners
+futurist
+greenlet
+idna
+#should we install this from git instead ?
+#becomes complicated downstream though
+ironic
```

Comment:
Would we pickup last release in that case downstream? I suspect that might be okay although I suspect we need some more flexibility upstream than we do downstream with brand new features.

Reply:
we can specify the version we need, upstream is already doable as we're using pip
downstream is early WIP